wallet-js/LICENSE_APACHE

```diff
@@ -0,0 +1,176 @@
+                              Apache License
```

Comment:
This is not the apache notice file to distribute, use the same one as in the other crates of this repo please

Reply:
done, I also replaced the MIT one just so they are all the same